### ISSUE TITLE
Link Github Repo to Jira 

### ISSUE THREAD

**Issue by sukanya-rath** [BCDevOps/devops-requests#1729]
## Step 0
**Do you have repos on bcgov GitHub Organization?** `Yes`
If yes, please proceed with the request.

**If there is similar applications/tools available in the community, have you tried them out?** `Yes`
There are many well supported community tools for various purposes, such SonarCube and BDDStack. 
Before you start with the new tools, have you consider trying out those that are available and widely adopted by the other application teams? 
If you have done so and do found that the existing ones does not match your need, please proceed.

**Have you tested out this GitHub application already?** `Yes`
We understand that there are a lot of new and cool GitHub Applications out there that teams could like to try out. 
However, we cannot just enable the integration without evaluating the app, as those are open to all application teams. 
We believe you have tested out the GitHub apps and had good reasoning for the preference before making this request.

## Step 1
Please fill out the team and project info so we know how to contact you:

* Product Name: TBS-BTF
* Team Name: FIN-IMB-AMD
* GitHub Repositories on bcgov: https://github.com/bcgov/fin-tbs-btf

## Step 2
Please provide details about the requested GitHub App. Take a look here for a [good example](https://github.com/BCDevOps/platform-services/issues/120/)

1. Brief introduction of the app: Would like the Github Repo to be connected with Jira for the application . 
2. Jira Link : https://finrms.atlassian.net/jira/software/projects/BTF/boards/78

3. General Criteria:
  - How Easy / Fast / Repeatable is it to use and maintain for both Development Teams and Developer Experience Teams:
  - Customizable:
  - User Access Model:
  - License:
  - BCGov STRA:
  - Price Model:

4. Comparison between Existing Tools Used by the Community
  - pros:
  - cons:

5. Relevant Resources and Links
6.

**Comment by MonicaG** [comment]
Hi,

I've added the Jira GitHub app to the  https://github.com/bcgov/fin-tbs-btf repo.

However, there are a few steps to complete before setup is complete.

Please have your team’s Jira administrator do the following steps:

* Install the “[GitHub for Jira app](https://bcdevex.atlassian.net/jira/marketplace/discover/app/com.github.integration.production)” into Jira (if it isn’t already).
   * Refer to the “Install the GitHub for Jira app” section in [Jira’s instructions on how to install ](https://support.atlassian.com/jira-cloud-administration/docs/integrate-with-github/)the GitHub App.
* After the app is installed, connect it to the bcgov organization
   * Refer to the “Connect a GitHub organization” section in [Jira’s instruction](https://support.atlassian.com/jira-cloud-administration/docs/integrate-with-github/)s on how to connect the org.
   * The process will give  a link that needs to be clicked on by a bcgov owner. That is the Developer Experience team. Have the administrator send the link to the team at Developer.Experience@gov.bc.ca We will click the link. That will finish the setup process.


Once the integration is complete, you won't see your data showing up right away as Jira needs to perform its[ backfill process](https://support.atlassian.com/jira-cloud-administration/docs/understand-github-for-jiras-initial-backfill-process/). The backfill defaults to 6 months but you can change that time frame if you want in the Settings section of the GitHub app in jira. It usually takes a few hours for the backfill process to complete.

**Comment by MonicaG** [comment]
Hi,
I'm closing this issue. Please feel free to reopen or create a new issue if anything isn't working.

Cheers,
Monica
